### PR TITLE
nodePackages.standard: init at 17.1.0

### PR DESCRIPTION
**Pull request by god464**
## Description of changes
https://github.com/standard/standard
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

## P.S.
Due to my mishandling of [253279](https://github.com/NixOS/nixpkgs/pull/253279), it was possible to ping other unwanted people, and I don't know how to undo that action. So I decided to close that PR and reopen one.

### CODE REVIEW

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please ask upstream to add a lock file and use e.g. `buildNpmPackage`. See #229475.

File: pkgs/development/tools/standard/package-lock.json
Comment (dotlambda) [inline pkgs/development/tools/standard/package-lock.json:1]: Vendoring 7k lines for a package is not acceptable

File: pkgs/development/node-packages/aliases.nix [hunk 1]
```diff
@@ -116,5 +116,6 @@ mapAliases {
   inherit (pkgs) write-good; # added 2023-08-20
   inherit (pkgs) yaml-language-server; # added 2023-09-05
   inherit (pkgs) yo; # added 2023-08-20
+  inherit (pkgs) standard; # added 2023-09-10
```
Comment (dotlambda) [inline pkgs/development/node-packages/aliases.nix:119]: No need for this if `nodePackages.standard` never existed.
```suggestion
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/standard/package-lock.json
Comment (god464) [reply pkgs/development/tools/standard/package-lock.json:1]: I don't have any good idea for this unless upstream is willing to provide `package-lock.json`.

Comment (dotlambda) [reply pkgs/development/tools/standard/package-lock.json:1]: You should ask them to do so. Better yet, make an upstream PR.

Comment (god464) [reply pkgs/development/tools/standard/package-lock.json:1]: > You should ask them to do so. Better yet, make an upstream PR.

Emm, how can I do for [this](https://github.com/standard/standard/blob/master/.npmrc#L1)？

Comment (dotlambda) [reply pkgs/development/tools/standard/package-lock.json:1]: I guess you have to remove that file. In your PR/issue you should argue why having a lock file is worth it.

**Pull request closed without merging**